src/pages/editor.tsx

```diff
@@ -19,7 +19,7 @@ export default function EditorPageContainer(props: PageProps): JSX.Element {
     setHasMounted(true);
   }, []);
   if (!hasMounted) {
-    return null;
+    return <></>;
```

Comment:
It may be better to modify the return type of `EditorPageContainer` to include `null`. Not a big deal though